[BUG]-EAE not editable

edit active effects throws an error:

foundry.mjs:26887 Uncaught (in promise) Error: Failed to render Application "EarthdawnActiveEffectSheet-Item-Wr8x3BImhZAsitfW-ActiveEffect-XMGBxgW886e3qKC7":
Cannot use 'in' operator to search for 'header' in undefined
    at #render (foundry.mjs:26887:13)
    at async #try (foundry.mjs:7164:17)Caused by: TypeError: Cannot use 'in' operator to search for 'header' in undefined
    at EarthdawnActiveEffectSheet._preparePartContext (foundry.mjs:79391:17)
    at async EarthdawnActiveEffectSheet._preparePartContext (eae-sheet.mjs:165:25)
    at async EarthdawnActiveEffectSheet._renderHTML (foundry.mjs:28213:29)
    at async #render (foundry.mjs:26878:22)
    at async #try (foundry.mjs:7164:17)

--- a/module/applications/effect/eae-sheet.mjs
+++ b/module/applications/effect/eae-sheet.mjs
@@ -50,8 +50,9 @@ export default class EarthdawnActiveEffectSheet extends ActiveEffectConfig {
         ...ActiveEffectConfig.TABS.sheet.tabs,
         { id: "execution", icon: ED4E.icons.effectExecution },
       ],
+      labelPrefix: "ED.Sheet.Tabs",
     },
-    labelPrefix: "ED.Sheet.Tabs",
+    
   };
 
   // region Properties

